src/iocore/net/libinknet_stub.cc

```diff
@@ -21,119 +21,6 @@
   limitations under the License.
  */
 
-class EThread;
-class Continuation;
+#include "tscore/Version.h"
 
-#include "P_UnixNet.h"
-#include "../dns/P_DNSConnection.h"
-int
-DNSConnection::close()
-{
-  ink_assert(false);
-  return 0;
-}
-
-void
-DNSConnection::trigger()
-{
-  ink_assert(false);
-}
-
-#include "proxy/StatPages.h"
-void
-StatPagesManager::register_http(char const *, Action *(*)(Continuation *, HTTPHdr *))
-{
-  ink_assert(false);
-}
-
-#include "proxy/ParentSelection.h"
-void
-SocksServerConfig::startup()
-{
-  ink_assert(false);
-}
-
-int SocksServerConfig::m_id = 0;
-
-void
-ParentConfigParams::findParent(HttpRequestData *, ParentResult *, unsigned int, unsigned int)
-{
-  ink_assert(false);
-}
-
-void
-ParentConfigParams::nextParent(HttpRequestData *, ParentResult *, unsigned int, unsigned int)
-{
-  ink_assert(false);
-}
-
-#include "api/InkAPIInternal.h"
-#include "proxy/ControlMatcher.h"
-char *
-HttpRequestData::get_string()
-{
-  ink_assert(false);
-  return nullptr;
-}
-
-const char *
-HttpRequestData::get_host()
-{
-  ink_assert(false);
-  return nullptr;
-}
-
-sockaddr const *
-HttpRequestData::get_ip()
-{
-  ink_assert(false);
-  return nullptr;
-}
-
-sockaddr const *
-HttpRequestData::get_client_ip()
-{
-  ink_assert(false);
-  return nullptr;
-}
-
-StatPagesManager statPagesManager;
-
-#include "api/FetchSM.h"
-ClassAllocator<FetchSM> FetchSMAllocator("unusedFetchSMAllocator");
-void
-FetchSM::ext_launch()
-{
-}
-void
-FetchSM::ext_destroy()
-{
-}
-ssize_t
-FetchSM::ext_read_data(char *, unsigned long)
-{
-  return 0;
-}
-void
-FetchSM::ext_add_header(char const *, int, char const *, int)
-{
-}
-void
-FetchSM::ext_write_data(void const *, unsigned long)
-{
-}
-void *
-FetchSM::ext_get_user_data()
-{
-  return nullptr;
-}
-void
-FetchSM::ext_set_user_data(void *)
-{
-}
-void
-FetchSM::ext_init(Continuation *, char const *, char const *, char const *, sockaddr const *, int)
-{
-}
-
-ChunkedHandler::ChunkedHandler() {}
+AppVersionInfo appVersionInfo;
```

Comment:
Why is this change in this PR?

Reply:
The net_test had a bunch of missing symbol issues with changing things into iocore/net. This was addressed by updating the library dependencies to avoid symbol issues, but now AppVersionInfo needed to be stubbed.